### PR TITLE
add: monthName

### PR DESCRIPTION
**Pull request by Aftabul-islam**

### CODE REVIEW

File: react-clock/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "bootstrap": "^4.6.0",
+    "hijri-js": "^1.0.25",
```
Comment (kabirahasaan) [inline react-clock/package.json:15]: I guess you need to be in sync with the main branch, this package should already be in there.

File: react-clock/src/calendar/index.tsx [hunk 2]
```diff
@@ -10,19 +11,26 @@ const formateHijriDate = (date : any) =>{
...
+    let hijriDate = hijriConverter.toHijri(moment(date).format('DD-MM-YYYY'), '-');
+    return `${hijriDate.monthName}`;
+};
 
 const Calendar = () => {
   let date = new Date();
   let formatedDate = moment(date).format('YYYY-MM-DD');
   const [currentDate, setCurrentDate] = useState(formatedDate);
   const [currentHijiriDate, setHijiriCurrentDate] = useState(formateHijriDate(date));
+  const [cradHijriMonth] = useState(formatCardHijrimonth(date));
```
Comment (kabirahasaan) [inline react-clock/src/calendar/index.tsx:27]: You do not need a state if you aren't tracking the changes on the fly

File: react-clock/yarn.lock [hunk 3]
```diff
@@ -5567,6 +5567,11 @@ hex-color-regex@^1.1.0:
   resolved "https://registry.yarnpkg.com/hex-color-regex/-/hex-color-regex-1.1.0.tgz#4c06fccb4602fe2602b3c93df82d7e7dbf1a8a8e"
   integrity sha512-l9sfDFsuqtOqKDsQdqrMRk0U85RZc0RtOR9yPI7mRVOa4FsR/BVnZ0shmQRM96Ji99kYZP/7hn1cedc1+ApsTQ==
 
+hijri-js@^1.0.25:
```
Comment (kabirahasaan) [inline react-clock/yarn.lock:5570]: be in sync with the Main branch

File: react-clock/src/calendar/index.tsx (hunk 2 above)
Comment (Aftabul-islam) [reply react-clock/src/calendar/index.tsx:27]: oh then i will remove this

**Pull request closed without merging**